src/ts/helpers/models/services.ts

```diff
@@ -1,48 +1,50 @@
 import i from '../../i18n';
 
-export default [
-  {
-    key: 'parkingGarage',
-    label: i('Parking/Garage'),
-  },
-  {
-    key: 'recommendedMechanic',
-    label: i('Recommended Mechanic'),
-  },
-  {
-    key: 'transportTheMotorcycle',
-    label: i('Transport The Motorcycle'),
-  },
-  {
-    key: 'transportToFromMotorcycle',
-    label: i('Transport To/From'),
-  },
-  {
-    key: 'photographer',
-    label: i('Photographer'),
-  },
-  {
-    key: 'mechanicalChangesOptions',
-    label: i('Mechanical Changes Options'),
-  },
-  {
-    key: 'guide',
-    label: i('Guide'),
-  },
-  {
-    key: 'coach',
-    label: i('Coach'),
-  },
-  {
-    key: 'group',
-    label: i('Group'),
-  },
-  {
-    key: 'club',
-    label: i('Club'),
-  },
-  {
-    key: 'nearbyCircuitTrail',
-    label: i('Nearby Circuit/Trail'),
-  },
-];
+export default function () {
+  return [
+    {
+      key: 'parkingGarage',
+      label: i('Parking/Garage'),
+    },
+    {
+      key: 'recommendedMechanic',
+      label: i('Recommended Mechanic'),
+    },
+    {
+      key: 'transportTheMotorcycle',
+      label: i('Transport The Motorcycle'),
+    },
+    {
+      key: 'transportToFromMotorcycle',
+      label: i('Transport To/From'),
+    },
+    {
+      key: 'photographer',
+      label: i('Photographer'),
+    },
+    {
+      key: 'mechanicalChangesOptions',
+      label: i('Mechanical Changes Options'),
+    },
+    {
+      key: 'guide',
+      label: i('Guide'),
+    },
+    {
+      key: 'coach',
+      label: i('Coach'),
+    },
+    {
+      key: 'group',
+      label: i('Group'),
+    },
+    {
+      key: 'club',
+      label: i('Club'),
+    },
+    {
+      key: 'nearbyCircuitTrail',
+      label: i('Nearby Circuit/Trail'),
```

Comment:
what is your opinion on the performance of calling `i()` on every element, at this point?